### PR TITLE
[14.0][mig]web_widget_domain_editor_dialog

### PR DESCRIPTION
**Pull request by hkapatel-initos**

### CODE REVIEW

**Comment by hkapatel-initos** [comment]
it would be nice if someone add reviews here.

File: web_widget_domain_editor_dialog/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Web Widget Domain Editor Dialog",
     "summary": "Recovers the Domain Editor Dialog functionality",
-    "version": "13.0.1.0.2",
+    "version": "14.0.1.0.2",
```
Comment (dsolanki-initos) [inline web_widget_domain_editor_dialog/__manifest__.py]: Update version to 14.0.1.0.0

Comment (hkapatel-initos) [reply web_widget_domain_editor_dialog/__manifest__.py]: > Update version to 14.0.1.0.0

Done

**Comment by hkapatel-initos** [comment]
@chienandalu it would be nice if you add a review here.

**Comment by hkapatel-initos** [comment]
@chienandalu and @JordiBForgeFlow  it would be nice if you both add a review here.

**Review by chienandalu — APPROVED** [review]
Tested on runbot. Thanks!

**Pull request merged**